src/View/UIView.php

```diff
@@ -17,12 +17,14 @@ class UIView extends View
     /**
      * Initialization hook method.
      *
+     * @param array $options
      * @return void
      */
-    public function initialize()
+    public function initialize(array $options = [])
```

Comment:
then we can just use the trait for the custom layout, and the class will always use the plugin's one